.github/renovate.json5

```diff
@@ -38,8 +38,10 @@
       ]
     },
     {
+      // Update grafana-build-tools tags across the repository.
       "customType": "regex",
-      "datasourceTemplate": "github-tags",
+      "depNameTemplate": "ghcr.io/grafana/grafana-build-tools",
+      "datasourceTemplate": "docker",
```

Comment:
It's OK for me either way, but why is this better?

Reply:
I don't think it is better, it should work the same. It is perhaps a bit more obvious. When I first saw this I wondered: Is there a reason to not do the obvious thing and look at the OCI registry for OCI image tags?

Reply:
The reason is that I was trying to test the configuration before actually committing anything to the repo, and with docker it was failing in even less useful ways than with github-tags.

I *do* think docker is the right thing to do here, because in this case the tag might be published a long time _before_ the docker image is available.

Reply:
That's interesting, I'll follow up on that to see why that might happen.